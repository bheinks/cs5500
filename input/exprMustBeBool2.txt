(* 
   Expression must be of type boolean
*)

program p;

var i : integer;
    c : char;

begin
  while c do
  begin
   i := i + 1;
   write(i)
  end;
  write('!')
end.

