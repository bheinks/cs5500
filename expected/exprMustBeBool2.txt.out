Line 11: Expression must be of type boolean
